File: 김태일/211118/약수의개수와덧셈.py
def isPrime (num):
    count = 0
    for i in range(1, num + 1):
        if num % i == 0:
            count += 1
    return count
        

def solution(left, right):
    answer = 0
    for i in range(left, right + 1):
        count = isPrime(i)
        print(i, count)
        if count % 2 == 0:
            answer += i
        else:
            answer -= i
    return answer

File: 유소연/20211118/약수의 개수와 덧셈.js
function solution(left, right) {
  let result = 0;
  for (let i = left; i <= right; i++) {
      result = Math.sqrt(i) % 1 === 0 ? result - i : result + i;
  }
  return result;
}

// const divisorCnt = (n) => {
//   let count = 0;
//   for (let i = 1; i * i <= n; i++) {
//       if (n % i === 0) {
//           count = i * i === n ? count + 2 : count + 1;
//       } 
//   }
//   return count;
// }